tests/fixtures/bigquery_fixtures.py

```diff
@@ -0,0 +1,304 @@
+from fidesops.models.datasetconfig import DatasetConfig
+from fidesops.schemas.connection_configuration import BigQuerySchema
+from fidesops.service.connectors import BigQueryConnector, get_connector
+from .application_fixtures import integration_config
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture(scope="function")
+def bigquery_connection_config(db: Session) -> Generator:
+    connection_config = ConnectionConfig.create(
+        db=db,
+        data={
+            "name": str(uuid4()),
+            "key": "my_bigquery_config",
+            "connection_type": ConnectionType.bigquery,
+            "access": AccessLevel.write,
+        },
+    )
+    # Pulling from integration config file or GitHub secrets
+    keyfile_creds = integration_config.get("bigquery", {}).get("keyfile_creds") or os.environ.get(
+        "BIGQUERY_KEYFILE_CREDS"
+    )
+    dataset = integration_config.get("bigquery", {}).get(
+        "dataset"
+    ) or os.environ.get("BIGQUERY_DATASET")
+    if keyfile_creds:
+        schema = BigQuerySchema(keyfile_creds=keyfile_creds, dataset=dataset)
+        connection_config.secrets = schema.dict()
+        connection_config.save(db=db)
+
+    yield connection_config
+    connection_config.delete(db)
+
+
+@pytest.fixture
+def bigquery_example_test_dataset_config(
+        connection_config: ConnectionConfig,
+        db: Session,
+        example_datasets: List[Dict],
+) -> Generator:
+    bigquery_dataset = example_datasets[6]
+    fides_key = bigquery_dataset["fides_key"]
+    connection_config.name = fides_key
+    connection_config.key = fides_key
+    connection_config.save(db=db)
+    dataset = DatasetConfig.create(
+        db=db,
+        data={
+            "connection_config_id": connection_config.id,
+            "fides_key": fides_key,
+            "dataset": bigquery_dataset,
+        },
+    )
+    yield dataset
+    dataset.delete(db=db)
+
+
+@pytest.fixture(scope="session")
+def bigquery_test_engine() -> Generator:
+    """Return a connection to a Google BigQuery Warehouse"""
+
+    connection_config = ConnectionConfig(
+        name="My BigQuery Config",
+        key="test_bigquery_key",
+        connection_type=ConnectionType.bigquery,
+    )
+
+    # Pulling from integration config file or GitHub secrets
+    keyfile_creds = integration_config.get("bigquery", {}).get("keyfile_creds") or os.environ.get(
+        "BIGQUERY_KEYFILE_CREDS"
+    )
+    dataset = integration_config.get("bigquery", {}).get(
+        "dataset"
+    ) or os.environ.get("BIGQUERY_DATASET")
+    if keyfile_creds:
+        schema = BigQuerySchema(keyfile_creds=keyfile_creds, dataset=dataset)
+        connection_config.secrets = schema.dict()
+
+    connector: BigQueryConnector = get_connector(connection_config)
+    engine = connector.client()
+    yield engine
+    engine.dispose()
+
+
+def seed_bigquery_integration_db(bigquery_integration_engine) -> None:
+    """
+    Currently unused.
```

Comment:
I see this is currently unused but does it make sense to move these SQL statements under `/data/sql` in the future? Is that the directory we use for seed data?

Reply:
I'm open to it! Some caveats:

* Those SQL statement files in `/data/sql` are typically referenced by the corresponding `docker-compose.integration-[datastore].yml` files
* If we wanted to run this seed SQL, it will be more complex to load in and run the SQL via python, as opposed to if it were in a python method. Alternatively we could copy/paste the SQL directly in bigquery console, but only if the user has access to the console. I don't anticipating having to run this code often, but it's good to have an easy way to run it just in case.
* we do have other seed data in the corresponding `[datastore]-fixtures,py` files, so this pattern follows convention

Reply:
Thanks for the explanation, it makes sense to me now why it's here. Let's not overcomplicate things then and keep this where it is.